app.py

```diff
@@ -0,0 +1,14 @@
+from gevent.pywsgi import WSGIServer
```

Comment:
in general: use black, mypy, flake8 and isort - libraries that we use in pre commit